src/components/Eth2BeaconChainActions.js

```diff
@@ -93,29 +98,48 @@ const Eth2BeaconChainActions = ({ data }) => {
       <StyledCardContainer>
         <StyledCardLeft
           emoji=":money_with_wings:"
-          title="Become a staker"
-          description="Staking is live! If you want to stake your ETH to help secure the network, make sure you’re aware of the risks."
+          title={intl.formatMessage({
```

Comment:
For these uses of `intl.formatMessage` let's refactor to use our new utility function `translateMessageId `:
https://github.com/ethereum/ethereum-org-website/blob/dev/src/utils/translations.js#L56
Moving forward this will save us from the repetition of adding a `defaultMessage`.

More context on the reasoning here: https://github.com/ethereum/ethereum-org-website/pull/1940